deploy/sfsturbo-csi-plugin/kubernetes/csi-sfsturbo-node.yaml

```diff
@@ -53,7 +53,7 @@ spec:
         - name: sfsturbo
           image: zhenguo/sfsturbo-csi-plugin:latest
```

Comment:
pls change the image to swr.cn-north-4.myhuaweicloud.com/k8s-csi/sfs-turbo-csi-plugin:v0.1.0

Reply:
done